lib/i18n/backend/active_record/store_procs.rb

```diff
@@ -33,7 +33,9 @@ def value=(val)
           end
         end
 
-        Translation.send(:include, self) if method(:to_s).respond_to?(:to_ruby)
+        if method(:to_s).respond_to?(:to_ruby)
+          I18n::Backend::ActiveRecord.config.translation_model.send(:include, self)
```

Comment:
I know that this module is not used(see https://github.com/svenfuchs/i18n-active_record/issues/68)

But I would mix the `TranslationModel` module here as well, so we could use `translation_model.send...`

Reply:
Hmm, why not `include`?
Again, this module is not used, but still

Reply:
Makes sense 👍🏻